src/stdlib/match_datadog_query.rs

```diff
@@ -162,6 +162,29 @@ impl Filter<Value> for VrlFilter {
         })
     }
 
+    fn phrase(
+        &self,
+        field: Field,
+        phrase: &str,
+    ) -> Result<Box<dyn Matcher<Value>>, PathParseError> {
+        let buf = lookup_field(&field)?;
+        match field {
+            // Default fields are compared by word boundary.
+            Field::Default(_) => {
+                let re = word_regex(phrase);
+                Ok(resolve_value(
+                    buf,
+                    Run::boxed(move |value| match value {
+                        Value::Bytes(val) => re.is_match(&String::from_utf8_lossy(val)),
+                        _ => false,
+                    }),
+                ))
+            }
```

Comment:
How is this different from the condition in `equals` below. It looks word-for-word identical but maybe I missed something. I'd also like to see some unit test cases that demonstrate the different behavior.

Reply:
It is the same, since I didn't know the impact of the change, so currently it's a non-breaking behavior.
Currently the non-phrase behavior is also behaving incorrectly, they both would need to be adjusted to include tokenization (and probably some adaption on the query parser)

I implemented it partially here https://github.com/DataDog/pomsky/pull/79. For full compatibility I'd need to have a closer look on how percolation tokenizes.